docs/helm-broker/docs/03-04-bundles-docs.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Bundles docs
+type: Details
+---
+
+## Documentation for ServiceClasses provided by bundle
+
+The Helm Broker in the provisioning process pushes the `addonRepositoryURL` variable into installed chart. With this feature a bundle which provides its own ServiceClasses e.g. `Azure Broker` can also install a documentation for them.
+
+If the bundle provides ServiceClasses and you want to have a documentation for it, you should create a `docs.yaml` inside the bundle's chart with the following structure:
+```
+apiVersion: cms.kyma-project.io/v1alpha1
+kind: ClusterDocsTopic
+metadata:
+  labels:
+    cms.kyma-project.io/view-context: service-catalog
+  name: {ServiceClass ID}
+spec:
+  displayName: "Slack Connector Add-on"
+  description: "Overall documentation, OpenAPI and AsyncAPI for Slack Connector Add-on"
+  sources:
+    - type: {type}
+      name: {name}  # must be unique within sources
+      mode: {mode}
+      url: {.Values.addonsRepositoryURL}
+      filter: docs/{class_name}/   # provide filter only if you use the package mode
+___
+apiVersion: cms.kyma-project.io/v1alpha1
+kind: ClusterDocsTopic
+metadata ... 
+```
+
```

Comment:
BUT! ;) I'm not sure if we should mention it at all if everything is described here: https://github.com/kyma-project/kyma/blob/master/docs/headless-cms/docs/06-02-clusterdocstopic.md.
Perhaps you could just make a reference to this doc and say sth like: `For detailed descriptions of all parameters, see the [ClusterDocsTopic custom resource](/components/headless-cms/#custom-resource-clusterdocstopic).` Hm?

Reply:
I already link that doc in the 03-01-create-bundles doc so I will leave like it is. Also, for now, the fields are the same but we can add there some new fields so it's better to leave that column.

Reply:
Actually, I agree with @kazydek. Instead of describing the fields here, make a reference to the relevant documentation. We don't know if and when we're going to add more fields.